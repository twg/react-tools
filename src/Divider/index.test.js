/* eslint-env jest */

import React from 'react'
import { Divider, VerticalDivider } from './'
import renderer from 'react-test-renderer'

describe('Divider', () => {
  it('matches snapshot', () => {
    let component = renderer.create(
      <div>
        <div />
        <Divider />
        <div />
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})

describe('VerticalDivider', () => {
  it('matches snapshot', () => {
    let component = renderer.create(
      <div>
        <span />
        <Divider />
        <span />
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
