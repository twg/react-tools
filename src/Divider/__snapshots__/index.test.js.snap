// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider matches snapshot 1`] = `
<div>
  <div />
  <div
    className="divider"
  />
  <div />
</div>
`;

exports[`VerticalDivider matches snapshot 1`] = `
<div>
  <span />
  <div
    className="divider"
  />
  <span />
</div>
`;
